### ISSUE TITLE
Welcome Forum Top Bar 404 Specifically From "Cannot Post Too Soon" Page

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#145]
![image](https://user-images.githubusercontent.com/52435234/63331164-17048080-c303-11e9-8951-e83b6d10e9bf.png)

redirects to here

![image](https://user-images.githubusercontent.com/52435234/63331181-1ec42500-c303-11e9-9941-902d8ed8e2a0.png)

`Chances are that https://www.roleplaygateway.com/posting.php?sid=7a94444bb1c1d13b5643c7e155a0e69a gave you a broken link. You should let them know so they can fix their link.`

**Comment by LawOfTheLand** [comment]
This looks like a bug regarding the interplay between login and redirect. You probably had the welcome desk pulled up, logged in from there, and you were redirected to the topic/forum with the session key in the URL. Then after you wrote your reply, the session key was in the redirect destination, which resulted in the 404. This is solved by logging in from the main forum page, then loading the Welcome Desk forum/topic, and then submitting a post.
